src/components/FilterChipsCarousel.tsx

```diff
@@ -23,28 +23,47 @@ export interface NetworkFilterChip<T> extends BaseFilterChip {
   selectedNetworkIds: NetworkId[]
 }
 
+export interface TokenSelectFilterChip<T> extends BaseFilterChip {
+  filterFn: (t: T, tokenId: string) => boolean
+  selectedTokenId: string
+}
+
 export function isNetworkChip<T>(chip: FilterChip<T>): chip is NetworkFilterChip<T> {
   return 'allNetworkIds' in chip
 }
 
-export type FilterChip<T> = BooleanFilterChip<T> | NetworkFilterChip<T>
+export function isTokenSelectChip<T>(chip: FilterChip<T>): chip is TokenSelectFilterChip<T> {
+  return 'selectedTokenId' in chip
+}
+
+export type FilterChip<T> = BooleanFilterChip<T> | NetworkFilterChip<T> | TokenSelectFilterChip<T>
 
 interface Props<T> {
   chips: FilterChip<T>[]
   onSelectChip(chip: FilterChip<T>): void
-  primaryColor: colors
-  secondaryColor: colors
+  primaryColor?: colors
+  primaryBorderColor?: colors
+  primaryTextColor?: colors
+  secondaryColor?: colors
+  secondaryBorderColor?: colors
+  secondaryTextColor?: colors
```

Comment:
can we drop these params if there's no usage of customizing? The less options the better. Also, would be nice if the styling change for this component is in a separate PR

Reply:
moved to https://github.com/valora-inc/wallet/pull/5672